packages/aws-cdk/lib/init-templates/v1/app/java/src/test/java/com/myorg/%name.PascalCased%Test.template.java

```diff
@@ -1,28 +1,25 @@
 package com.myorg;
 
 import software.amazon.awscdk.core.App;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.SerializationFeature;
-
+import software.amazon.awscdk.assertions.Template;
+import software.amazon.awscdk.assertions.Match;
 import java.io.IOException;
 
+import java.util.*;
+
 import org.junit.jupiter.api.Test;
-import static org.assertj.core.api.Assertions.assertThat;
 
 public class %name.PascalCased%Test {
-    private final static ObjectMapper JSON =
-        new ObjectMapper().configure(SerializationFeature.INDENT_OUTPUT, true);
 
     @Test
     public void testStack() throws IOException {
         App app = new App();
         %name.PascalCased%Stack stack = new %name.PascalCased%Stack(app, "test");
 
-        // synthesize the stack to a CloudFormation template
-        JsonNode actual = JSON.valueToTree(app.synth().getStackArtifact(stack.getArtifactId()).getTemplate());
+        Template template = Template.fromStack(stack);
 
         // Update once resources have been added to the stack
-        assertThat(actual.get("Resources")).isNull();
+        Map<String, Object> expected = Map.of();
+        template.templateMatches(expected);
```

Comment:
I've always looked at it as:

- `sample-app` template - creates a sample application with resources
- `app` template - creates an empty application

I don’t think we should add resources to the app template, otherwise what is the point of having both the sample-app and app templates? As a user I want to have a template that just gives me a blank application so I don’t have to go in and delete a bunch of resources before I start working.

Reply:
Decided to add an example resource along with an example test that are both commented out.